app/assets/javascripts/discourse/app/models/user.js

```diff
@@ -1436,6 +1436,10 @@ User.reopen(Evented, {
     this._subscribersCount--;
   },
 
+  isTrackingStatus() {
```

Comment:
would be very nice at some point to have all of this using glimmer and tracked properties, but too much scope creep for this PR

Reply:
Yes, if we have glimmer components everywhere, handling this will be simpler.